navigator_engine/common/graph_loader.py

```diff
@@ -310,6 +310,11 @@ def validate_pluggable_logic(node, network):
         for node, child_node in network.out_edges(node):
             if child_node.ref == function_args[0]:
                 arg_action_is_child = True
+            elif child_node.action:
+                # Check if child node is complete action
+                if child_node.action.complete:
+                    arg_action_is_child = True
+

```

Comment:
Not understanding why this is necessary... can you explain?

Reply:
I reviewed how manual confirmations work and this is actually not need in the end. Will remove the additional check.